auctionapp/src/main/java/com/internship/auctionapp/controller/ItemController.java

```diff
@@ -72,10 +76,18 @@ public List<ItemDto> getActiveSellerItems(@PathVariable(name = "id") UUID id) {
     public List<ItemDto> getSoldSellerItems(@PathVariable(name = "id") UUID id) {
         return itemService.getSoldSellerItems(id);
     }
+
     @GetMapping("/seller/bids/{id}")
     @PreAuthorize("#id == authentication.principal.id")
-    public List<ItemDto> getBiddedOnItemsByUser(@PathVariable(name = "id") UUID id){
+    public List<ItemDto> getBiddedOnItemsByUser(@PathVariable(name = "id") UUID id) {
         return itemService.getBiddedOnItemsByUser(id);
     }
+
+    @RequestMapping(value = "/{id}", method = RequestMethod.POST,
+            consumes = {"multipart/form-data"})
```

Comment:
- You should be able to use `PostMapping` annotation with `path` and `consumes`
- Use `MediaType.MULTIPART_FORM_DATA_VALUE`